### ISSUE TITLE
Grouped files and plugins

### ISSUE THREAD

**Issue by caclark** [BestImageViewer/geeqie#534]
### ISSUE TYPE

* Feature Request

### GEEQIE VERSION

```
All
```

### SUMMARY

Ref: https://sourceforge.net/p/geeqie/mailman/geeqie-devel/thread/1147172680.7660315.1509475406451%40mail.yahoo.com/#msg36098449

If a plugin is called from a set of grouped files, even if the %F parameter is used, only the first file name is passed over. As stated in the above ref., this gives a problem when trying to use external programs.

Although `geeqie -r --get-sidecars:$1 `  can be used to get the grouped files, there must be a better way.

**Comment by caclark** [comment]
The problem is that the code sees this group of files, and decides that the .desktop file needs only one (but I didn't check how it decides which one of the group).
I guess the code could be changed so that all files of the group, including sidecars, are sent to the .desktop file. But that means that all the files get opened in the editor - and no doubt that will create problems with other users.

I don't see a general-purpose solution to this.

**Comment by bdillahu** [comment]
A use case I have is a file rename script... would really like it to rename all the files in the group (different extensions get handled by the script)...

**Comment by caclark** [comment]
Can you use the --get-sidecars facility? e.g. in your script:

for file in "$@" ; do
    # we get only one file for each group, typically the main one
    # get the sidecars:
    geeqie -r --get-sidecars:"$file" |while read sidecar ; do
		# Do something with the each of the grouped files
		echo "$sidecar"
    done
done

**Comment by bdillahu** [comment]
Yes, thanks... that’s about what I wound up with after seeing that reference to the —get-sidecars option... did have to be careful with quoting to handle spaces in the name/path.
